packages/monorepo-scripts/src/publish.ts

```diff
@@ -107,30 +113,65 @@ package.ts. Please add an entry for it and try again.`,
     }
 }
 
+async function checkPublishRequiredSetupAsync(): Promise<boolean> {
+    // check to see if logged into npm before publishing
+    try {
+        await execAsync(`sudo npm whoami`);
+    } catch (err) {
+        utils.log('You must be logged into npm in the commandline to publish. Run `npm login` and try again.');
+        return false;
+    }
+
+    // Check to see if Git personal token setup
+    if (_.isUndefined(process.env.GITHUB_PERSONAL_ACCESS_TOKEN_0X_JS)) {
```

Comment:
Can we consolidate this and the githubPersonalAccessToken in postpublish_utils so that the environment variable exists in one place.